src/views/product-root-docs-path-landing/server.ts

```diff
@@ -158,27 +172,45 @@ const getStaticProps = async (context: GetStaticPropsContext) => {
 		marketingContentBlocksWithHeadingLevels
 	)
 
+	/**
+	 * Transform sidecarHeadings into outlineItems, and pageHeading
+	 */
+	const firstHeading = sidecarHeadings[0]
+	const pageHeading = { id: firstHeading.id, title: firstHeading.title }
```

Comment:
🙋🏻 Did pulling `pageHeading` out like this solve a particular problem? Just curious!

Reply:
Yep - broadly it carries forward an existing solution, we need to return `pageHeading` as part of the page props. 

From a more specific style perspective, I added a named `const` here primarily for clarity (it felt helpful to write this out in words, rather than assuming the meaning of `sidecarHeadings[0]` was clear), and also to avoid accessing the same array index twice. 

Hopefully that answers the question 🙇